README.md

```diff
@@ -1,21 +1,42 @@
 # Intel® Extension for DeepSpeed*
 Intel® Extension for DeepSpeed* is an extension that brings Intel GPU (XPU) support to DeepSpeed(https://github.com/Microsoft/DeepSpeed). It comes with the following components:
 1. DeepSpeed Accelerator Interface implementation
-2. DeepSpeed op builders implmentation for XPU
+2. DeepSpeed op builder implmentation for XPU
 3. DeepSpeed op builder kernel code
 
 DeepSpeed would automatically use Intel® Extension for DeepSpeed* when it is installed as a python package.   After installation, models ported for DeepSpeed Accelerator Interface that run on DeepSpeed could run on Intel GPU device.
 
-Usage:
+## Installation
+
+Optional: need to install [oneAPI Base Toolkit Packages](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html) when using Kernel Injection for inference
+
+`Intel® oneAPI DPC++/C++ Compiler` is required by OP builders to support the JIT build for Kernel Injection.
+
 1. Install Intel® Extension for DeepSpeed*
+```python
+pip install intel-extension-for-pytorch
+```
+ Or, you can build from source:
+```python
+source {DPCPPROOT}/env/vars.sh
```

Comment:
shoule be source ${DPCPPROOT}/env/vars.sh here?

Reply:
> shoule be source ${DPCPPROOT}/env/vars.sh here?

I follow the same convention from ipex installation guide https://intel.github.io/intel-extension-for-pytorch/xpu/latest/tutorials/installation.html

I guess the reason it is not using ${DPCPPROOT} probably because DPCPPROOT is not a predefined env variable. It is a place holder.

Reply:
Same illustration LGTM. (Just looks a little confused to use {} as place holder in CMD which may make user mixed with env variable.

Reply:
> Same illustration LGTM. (Just looks a little confused to use {} as place holder in CMD which may make user mixed with env variable.

let me add ${}. It looks better.